sagemaker-inference-deployment-guardrails/Update-SageMaker-Inference-endpoint-using-canary-traffic-shifting.ipynb

```diff
@@ -70,7 +70,15 @@
    "outputs": [],
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.    	metrics_all = invocation_metrics.join([metrics_epc_1], how="outer")
</pre><p>Formatting/style - looks like this notebook uses spaces but new revision uses tabs instead - switch to spaces?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3036/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>